README.md

```diff
@@ -118,24 +118,25 @@ docker compose -f docker-compose.yml -p memphis up
 
 ## Local access
 ### Via Kubernetes
-To access Memphis UI from localhost, run the below commands:
 ```shell
-kubectl port-forward service/memphis-ui 9000:80 --namespace memphis > /dev/null &
-```
+To access Memphis using UI/CLI/SDK from localhost, run the below commands:
 
-To access Memphis using CLI or SDK from localhost, run the below commands:</br>
-```shell
-kubectl port-forward service/memphis-cluster 6666:6666 5555:5555 --namespace memphis > /dev/null &
-```
-Dashboard: http://localhost:9000</br>
-Memphis broker: http://localhost:6666
+  - kubectl port-forward service/memphis-cluster 6666:6666 9000:9000 7770:7770 --namespace memphis > /dev/null &
 
+For interacting with the broker via HTTP:
+
+  - kubectl port-forward service/memphis-http-proxy 4444:4444 --namespace memphis > /dev/null &
+
+Dashboard/CLI: http://localhost:9000
+Broker: localhost:6666 (Client Connections)
+HTTP proxy: localhost:4444 (Data + Mgmt)
+```
 **For Production Environments**
 Please expose the UI, Cluster, and Control-plane via k8s ingress / load balancer / nodeport
 
 ### Via Docker
-UI - http://localhost:9000<br>
-Broker - http://localhost:6666<br>
+Dashboard/CLI: http://localhost:9000<br>
+Control-Plane - localhost:6666<br>
```

Comment:
this term is not in use anymore - please use broker instead

Reply:
Fixed